tests/test_tokenization.py

```diff
@@ -0,0 +1,89 @@
+import sys
+import os
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.dirname(SCRIPT_DIR))
+
+from gector.tokenization import *
+import unittest
+from transformers import AutoTokenizer
+
+class TokenizationTests(unittest.TestCase):
+    def test_get_bpe_groups(self):
+        token_offsets = [(0, 6), (7, 15), (16, 18), (19, 21), (22, 23), (24, 29), (30, 33), (34, 40), (41, 44), (45, 57), (58, 61), (62, 70), (71, 81), (82, 87), (88, 94), (95, 98), (99, 108), (109, 111), (112, 121), (122, 131), (132, 133)]
+        bpe_offsets = [(0, 6), (7, 15), (16, 18), (19, 21), (22, 23), (24, 29), (30, 33), (34, 40), (41, 44), (45, 54), (54, 57), (58, 61), (62, 70), (71, 74), (74, 81), (82, 87), (88, 94), (95, 98), (99, 108), (109, 111), (112, 121), (122, 131),(132, 133)]
+        input_ids = [50265, 37158, 15, 1012, 2156, 89, 32, 460, 5, 12043, 268, 8, 4131, 22761, 13659, 49, 1351, 8, 11360, 7, 12043, 7768, 479]
+        expected_results = ([[0], [1], [2], [3], [4], [5], [6], [7], [8], [9, 10], [11], [12], [13, 14], [15], [16], [17], [18], [19], [20], [21], [22]], [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22])
```

Comment:
Can we break up these very long lines? Usually, our best practice is to break lines longer than 101 characters.